src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Nsg-FlowLogs.json

```
@@ -1,15 +1,16 @@
 {
-  "name": "Deploy-Nsg-FlowLogs",
+  "name": "[Deprecated] Deploy-Nsg-FlowLogs",
```

```suggestion
  "name": "Deploy-Nsg-FlowLogs",
```

Only the display name gets changed as per https://github.com/Azure/azure-policy/blob/master/built-in-policies/README.md#preview-and-deprecated-policies

---

Didn't do it in `Deploy-Nsg-FlowLogs-to-LA.json`!!! I must've got "deprecation" happy 🤷‍♂️ amended and pushed